hooks/hookexecution/execution.go

```diff
@@ -311,3 +315,40 @@ func handleHookMutations[P any](
 
 	return payload
 }
+
+func handleModuleActivities[T any, P any](hook hooks.HookWrapper[T], activityControl privacy.ActivityControl, payload P) P {
+	// only 2 stages receive bidder request: hookstage.ProcessedAuctionRequestPayload and hookstage.BidderRequestPayload
+	// they both implement PayloadBidderRequest interface in order to execute mutations on bid request
+	var payloadData hookstage.PayloadBidderRequest
+	ok := true
+	if payloadData, ok = any(&payload).(hookstage.PayloadBidderRequest); !ok {
+		// payload doesn't have a bid request
+		return payload
+	}
+
+	privacyEnforcement := privacy.Enforcement{}
+
+	// parse hook.Module to split it to type and mame?
+	// hook.Module example: "mytest.mymodule". Can it be "rtd.mymodule" or "general.mymodule"?
```

Comment:
I think we use `mytest_mymodule` as the full module's name in other places.

Reply:
I'm confused here.
This is how hooks configuration looks like:
```json
 "hook_sequence": [
                    {
                      "module_code": "mytest.mymodule",
                      "hook_impl_code": "mymodule"
                    },
                    {
                      "module_code": "mytest2.mymodule2",
                      "hook_impl_code": "mymodule2"
                    }
                  ]
```
this is how activity configuration looks like:
```json
"rules": [
            {
              "allow": false,
              "condition": {
                "componentName": [
                  "general.mymodule2"
                ]
              }
            }
          ]
```
Do we expect to have activity component name set to `rtd.mytest.mymodule` or `general.mytest.mymodule`?
It will need an adjustment for activity rules parser, becaule we only expect to have one `.` in componentName.

Reply:
My thought is the `mytest` part is the same thing as `general` and `rtd`. It is just we only envision those two categories as being relevant at present. Do you have a different understanding of where the `mytest` piece of the name is coming from?

Reply:
Yes, this what I was thinking too. The problem here is that these two modules are created and configured by me and I can give any name to the directory where the module will be. Modules should be placed in {PBS_root}/modules directory. 

In order to make activities work this directories can only have names `rtd` and `general`. 

Maybe we should set module type to `hook_impl_code`, like 
```json
 {
    "module_code": "mytest.mymodule",
    "hook_impl_code": "rtd.mymodule"
 }
```